week02/group_anagrams.cpp

```diff
@@ -0,0 +1,33 @@
+// TC: O(10^4*100) = O(n*n)
+// MC:O(10^4*100) = O(n*n) for using vector of type vector of string
```

Comment:
What is `n` here? The time complexity analysis is so wrong that it can even drag you off the interview. Your solution and time complexity analysis are the opposite.

The correct analysis is `O(S)` where S = total number of characters in all the strings.